components/Footer/Footer.js

```diff
@@ -3,66 +3,146 @@ import Link from 'next/link';
 import classNames from 'classnames';
 import OutboundLink from 'components/_common_/OutboundLink/OutboundLink';
 import SocialMedia from 'components/SocialMedia/SocialMedia';
+import Button from 'components/_common_/Button/Button';
+import { s3 } from 'common/constants/urls';
 import styles from './Footer.css';
 
 function Footer() {
+  const currentYear = new Date().getFullYear();
+
   return (
     <footer className={styles.Footer}>
-      <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
-        <a className={styles.email} href="mailto:contact@operationcode.org">
-          contact@operationcode.org
-        </a>
-
-        <SocialMedia />
-      </div>
+      <div className={classNames(styles.footerWrapper, styles.row)}>
+        <div className={classNames(styles.footerGrouping, styles.socialGrouping)}>
+          <div className={classNames(styles.logoGrouping)}>
+            <img src={`${s3}branding/logos/small-blue-logo.png`} alt="Operation Code Logo" />
+          </div>
+          <div className={classNames(styles.capitalize, styles.marginBottom)}>Connect With Us!</div>
+          <SocialMedia />
+        </div>
 
-      <div className={classNames(styles.footerGrouping, styles.logoGrouping)}>
-        <img src="/static/images/icons/operation_code_logo.svg" alt="Operation Code Logo" />
+        <div className={classNames(styles.footerGrouping, styles.linksGrouping, styles.capitalize)}>
+          <div className={styles.linksRow}>
+            <div className={styles.linksColumn}>
+              <span>
+                <Link href="/about">
+                  <a>About Us</a>
+                </Link>
+              </span>
+              <span>
+                <Link href="/team">
+                  <a>Team</a>
+                </Link>
+              </span>
+              <span>
+                <Link href="/contact">
+                  <a>Contact Us</a>
+                </Link>
+              </span>
+              <span>
+                <Link href="/faq">
+                  <a>FAQ</a>
+                </Link>
+              </span>
+            </div>
+            <div className={styles.linksColumn}>
+              <span>
+                {' '}
```

Comment:
I was thinking about to modularize... okay let me work on this more. Ah missed that empty link 🤕

Reply:
I'm putting the footer links data in `/commons/constants/urls.js` -- Let me know if it this is the wrong place or if it's in the wrong format. You're right, it would be much easier to edit later on.